docs/changelog.md

```diff
@@ -48,6 +48,9 @@ permalink: /changelog/
     menu also contains an access entry for Add-ons Manager, for which `onAddonsManagerTapped` needs to be passed in the
     constructor.
 
+* **local development**
+  * Enable local Gradle Build Cache to speed-up local builds.
```

Comment:
Let's make sure everyone knows about this in case they run into errors and may need to nuke their cache.

Reply:
I'll send out an email after this lands.